src/main.rs

```diff
@@ -78,35 +73,44 @@ fn main() {
     let engine_config = &*config::ENGINE_CONFIG;
 
     let mut app = App::new();
-    app.insert_resource(WindowDescriptor {
-        title: "Fish Folk Punchy".to_string(),
-        scale_factor_override: Some(1.0),
-        ..default()
-    })
-    .insert_resource(ImageSettings::default_nearest());
-
```

Comment:
we need a `ImagePlugin::default_nearest()` somewhere in the DefaultPlugins builder to replace this so the filtering mode is correct.

Reply:
Somehow this got lost in the upgrade. I’ll quickly add this back. Nice catch 😅